src/algebra/lie/basic.lean

```diff
@@ -1222,6 +1222,9 @@ class lie_module.is_irreducible : Prop :=
 class lie_module.is_nilpotent : Prop :=
 (nilpotent : ∃ k, lie_module.lower_central_series R L M k = ⊥)
 
+lemma trivial_is_nilpotent [lie_module.is_trivial L M] : lie_module.is_nilpotent R L M :=
```

Comment:
this can be an instance, right?

Reply:
Yes, thanks!

I'll push that up with the next commit, which may or may not end up arriving tomorrow :-)